cotix/_visualizer_tools.py

```diff
@@ -0,0 +1,80 @@
+import abc
+
+import jax.numpy as jnp
+import pygame
+import pygame.locals
+
+from ._bodies import Ball
+
+
+class AbstractDrawer:
+    @abc.abstractmethod
+    def draw(self, window, pix2meter):
+        """
+        Draw the object (e.g. a body) in the selected pygame window at a selected
+        moment of time.
+        This will typically call pygame.draw() methods based on the properties of
+        the object
+        """
+
+        raise NotImplementedError
+
+
+class BallDrawer(AbstractDrawer):
+    object: Ball
+    color: pygame.Color
+    # the list of visualization properties will change depending on the object to draw
+
+    def __init__(self, obj: Ball, c: pygame.Color):
+        self.object = obj
+        self.color = c
+
+    def draw(self, window, pix2meter):
+        center_x, center_y = self.object.position[0], self.object.position[1]
+        # retrieve the first coord of the global support as it should be equal
+        # to the radius
+        # for more complex shapes we will need more info (get 360 points around
+        # a circle with the global_support?)
+        radius = self.object.shape.get_global_support(jnp.array([1.0, 0.0]))[0]
+
+        # convert meters to pixels with scale
+        center_x *= pix2meter
+        center_y *= pix2meter
+        radius *= pix2meter
```

Comment:
ins't meter to pix a better name? Since you want to convert meters to pixels

Reply:
True, my bad! I was thinking about `pix_by_meter` but I end up writing other thing